api/envoy/admin/v2alpha/config_dump.proto

```diff
@@ -47,6 +47,9 @@ message UpdateFailureState {
 
   // Details about the last failed update attempt.
   string details = 3;
+
+  // This is the version of the rejected resource.
+  string version_info = 4;
```

Comment:
There's no need to make changes in v2alpha at this point.  Let's just make the change in v3.